test/test-parse-var.cpp

```diff
@@ -1802,13 +1804,18 @@ TEST(test_parse, variables_can_be_named_contextual_keywords) {
                   ElementsAre(spy_visitor::visited_variable_use{name}));
     }
 
-    // TODO(#215): Disallow 'await' as parameter to async arrow function.
     for (string8 code : {
              u8"(async " + name + u8" => null)",
              u8"(async (" + name + u8") => null)",
              u8"(" + name + u8" => null)",
              u8"((" + name + u8") => null)",
          }) {
+      if (name == u8"await" &&
```

Comment:
Should this change land as it's own commit before all the other changes?

Reply:
Nah, this is fine as-is.